### ISSUE TITLE
Does not work on Arch Linux, GNOME 3.15.91

### ISSUE THREAD

**Issue by ghost** [johnhoran/Laine#3]
Arch Linux current with [gnome-unstable] enabled.

It does not throw any errors, but not any sign of it working either.

Maybe this does not work with Arch Linux - here is [PKGBUILD](https://projects.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/pulseaudio) for pulseaudio that was used.

**Comment by johnhoran** [comment]
I'll set up a VM when I get a chance and take a look.  I have tried it out in Fedora 22, which is also on 3.15.91, and it seems to work there.  I use Arch on my main machine, but I generally don't track any of the unstable branches.  

Anyway thanks for the bug report.

**Comment by johnhoran** [comment]
The extension isn't marked as compatible with 3.15.91, so like most extensions it won't load from the website.  It does actually work with 3.15.91, but as it often takes a while for the website to review changes generally extension makers don't target development versions of gnome.  It is possible to install most extensions manually, which is what I did to test this.

So this will be fixed when 3.16 is officially launched.

**Comment by ghost** [comment]
I just updated and everything works fine after reboot.

```
[2015-03-18 18:03] [ALPM] transaction started
[2015-03-18 18:03] [ALPM] upgraded btrfs-progs (3.18.2-1 -> 3.19-1)
[2015-03-18 18:04] [ALPM] upgraded clutter-gst (2.0.14-1 -> 3.0.4-1)
[2015-03-18 18:04] [ALPM] upgraded gnome-user-share (3.14.1-1 -> 3.14.2-1)
[2015-03-18 18:04] [ALPM] upgraded libnautilus-extension (3.14.2-2 -> 3.15.91-1)
[2015-03-18 18:04] [ALPM] upgraded libvdpau (0.9-1 -> 1.1-1)
[2015-03-18 18:04] [ALPM] upgraded libxfont (1.5.0-1 -> 1.5.1-1)
[2015-03-18 18:04] [ALPM] upgraded nautilus (3.14.2-2 -> 3.15.91-1)
[2015-03-18 18:04] [ALPM] upgraded patch (2.7.4-1 -> 2.7.5-1)
[2015-03-18 18:04] [ALPM] upgraded system-config-printer (1.5.5-2 -> 1.5.6-1)
[2015-03-18 18:04] [ALPM] upgraded totem (3.14.2-1 -> 3.15.91-1)
[2015-03-18 18:04] [ALPM] upgraded vino (3.14.1-1 -> 3.15.91-1)
[2015-03-18 18:04] [ALPM] upgraded yelp (3.14.1-1 -> 3.15.91-1)
[2015-03-18 18:04] [ALPM] transaction completed
```

Not sure why this not worked before, sorry for spending your time on this.

I really like how your extension is minimal in it's user interface and to the point.
